detect_x86.go

```diff
@@ -36,6 +36,56 @@ func addInfo(c *CPUInfo, safe bool) {
 	c.AVX10Level = c.supportAVX10()
 	c.cacheSize()
 	c.frequencies()
+	if c.maxFunc >= 0x0A {
+		eax, ebx, _, edx := cpuid(0x0A)
+		c.PMU = parseLeaf0AH(eax, ebx, edx)
+	}
 }
 
 func getVectorLength() (vl, pl uint64) { return 0, 0 }
+
+func parseLeaf0AH(eax, ebx, edx uint32) PerformanceMonitoringInfo {
```

Comment:
Maybe move to end of cpuid.go, so it can be picked up by tests.

Reply:
Done in 3b8bce2